### ISSUE TITLE
"Output file path does not exist" error

### ISSUE THREAD

**Issue by M0shr00m** [slhck/ffmpeg-normalize#135]
**Expected behavior**
ffmpeg-normalize sees an existing directory as existing

**Actual behavior**
"Output file path does not exist" error is returned, file is skipped

**Command**
The exact command you were trying to run:

```
ffmpeg-normalize "$file" -o "$file" --force --progress -c:a aac -b:a 350k --target-level -20 --debug --dual-mono
```

The full script I'm running:

```
find . -iregex '.*\.\(mkv\|mp4\|avi\)' -mindepth 1 -maxdepth 2 | while read line; do ffmpeg-normalize "$line" -o "$line" --force --progress -c:a aac -b:a 350k --target-level -20 --debug --dual-mono; done
```

**Environment (please complete the following information):**
- [ Ubuntu 1804 ] Your operating system
- [ 2.7.17 ] Your Python version / distribution (`python --version`)
- [ 3.4.8 ] Your ffmpeg version (`ffmpeg -version`)

*Note - I am running ffmpeg-normalize command in a do/while loop inside a bash script, in conjunction with the "find" command.  Every other file returns the "Output File path does not exist" error, and the other files succeed. It's consistently reproducible; odd numbered files succeed, even number files fail with this error.

**Comment by slhck** [comment]
Hmm, this is weird. When I run your loop, I can reproduce the behavior. When I run the commands printed by the loop one after the other, everything works fine.

**Comment by slhck** [comment]
So to be clear, when I do:

```
➜ find . -name '*.wav' -mindepth 1 -maxdepth 1 -print0 | while IFS= read -r -d $'\0' line; do echo ffmpeg-normalize "$line" -o "$line" --force; done
ffmpeg-normalize ./test4.wav -o ./test4.wav --force
ffmpeg-normalize ./test2.wav -o ./test2.wav --force
ffmpeg-normalize ./test3.wav -o ./test3.wav --force
ffmpeg-normalize ./test1.wav -o ./test1.wav --force
```

This looks okay.

However, when I actually run it and debug the CLI arguments:

```
➜ find . -name '*.wav' -mindepth 1 -maxdepth 1 -print0 | while IFS= read -r -d $'\0' line; do ffmpeg-normalize "$line" -o "$line" --force; done
Namespace(audio_bitrate=None, audio_codec=None, chapters_disable=False, debug=False, dry_run=False, dual_mono=False, extension='mkv', extra_input_options=None, extra_output_options=None, force=True, input=['./test4.wav'], keep_original_audio=False, loudness_range_target=7.0, metadata_disable=False, normalization_type='ebu', offset=0.0, output=['./test4.wav'], output_folder='normalized', output_format=None, post_filter=None, pre_filter=None, print_stats=False, progress=False, quiet=False, sample_rate=None, subtitle_disable=False, target_level=-23.0, true_peak=-2.0, verbose=False, video_codec='copy', video_disable=False)
WARNING: Audio stream has a duration of less than 3 seconds. Normalization may not work. See https://github.com/slhck/ffmpeg-normalize/issues/87 for more info.
WARNING: The sample rate will automatically be set to 192 kHz by the loudnorm filter. Specify -ar/--sample-rate to override it.
Namespace(audio_bitrate=None, audio_codec=None, chapters_disable=False, debug=False, dry_run=False, dual_mono=False, extension='mkv', extra_input_options=None, extra_output_options=None, force=True, input=['/test2.wav'], keep_original_audio=False, loudness_range_target=7.0, metadata_disable=False, normalization_type='ebu', offset=0.0, output=['/test2.wav'], output_folder='normalized', output_format=None, post_filter=None, pre_filter=None, print_stats=False, progress=False, quiet=False, sample_rate=None, subtitle_disable=False, target_level=-23.0, true_peak=-2.0, verbose=False, video_codec='copy', video_disable=False)
ERROR: file /test2.wav does not exist
Namespace(audio_bitrate=None, audio_codec=None, chapters_disable=False, debug=False, dry_run=False, dual_mono=False, extension='mkv', extra_input_options=None, extra_output_options=None, force=True, input=['./test3.wav'], keep_original_audio=False, loudness_range_target=7.0, metadata_disable=False, normalization_type='ebu', offset=0.0, output=['./test3.wav'], output_folder='normalized', output_format=None, post_filter=None, pre_filter=None, print_stats=False, progress=False, quiet=False, sample_rate=None, subtitle_disable=False, target_level=-23.0, true_peak=-2.0, verbose=False, video_codec='copy', video_disable=False)
WARNING: Audio stream has a duration of less than 3 seconds. Normalization may not work. See https://github.com/slhck/ffmpeg-normalize/issues/87 for more info.
WARNING: The sample rate will automatically be set to 192 kHz by the loudnorm filter. Specify -ar/--sample-rate to override it.
Namespace(audio_bitrate=None, audio_codec=None, chapters_disable=False, debug=False, dry_run=False, dual_mono=False, extension='mkv', extra_input_options=None, extra_output_options=None, force=True, input=['/test1.wav'], keep_original_audio=False, loudness_range_target=7.0, metadata_disable=False, normalization_type='ebu', offset=0.0, output=['/test1.wav'], output_folder='normalized', output_format=None, post_filter=None, pre_filter=None, print_stats=False, progress=False, quiet=False, sample_rate=None, subtitle_disable=False, target_level=-23.0, true_peak=-2.0, verbose=False, video_codec='copy', video_disable=False)
ERROR: file /test1.wav does not exist
```

You can see that in every other command run, the input is missing the `.`, so it's `input=['/test2.wav']` instead of `input=['./test2.wav']`.

I am not sure where the problem lies but I don't think it's my script?

**Comment by slhck** [comment]
When I contruct a simple command line interface with Python, it is not reproducible:

```
find . -name '*.wav' -mindepth 1 -maxdepth 1 -print0 | while IFS= read -r -d $'\0' line; do
        python3 -c "import argparse; parser = argparse.ArgumentParser(); parser.add_argument('input',nargs='+'); parser.add_argument('-o', '--output', nargs='+'); parser.add_argument('-f', '--force', action='store_true'); print(parser.parse_args());" "$line" -o "$line" --force
done
Namespace(force=True, input=['./test4.wav'], output=['./test4.wav'])
Namespace(force=True, input=['./test2.wav'], output=['./test2.wav'])
Namespace(force=True, input=['./test3.wav'], output=['./test3.wav'])
Namespace(force=True, input=['./test1.wav'], output=['./test1.wav'])
```

So I don't really know what is causing every *other* input string to become invalid.

**Comment by slhck** [comment]
Finally, when I do:

```
for line in *.wav; do ffmpeg-normalize "$line" -o "$line" --force; done
```

Everything works as expected. So it has something to do with the `while` loop.

**Comment by slhck** [comment]
Fixed by #138 and released just now in v1.20.2.
